tools/pioasm/go_output.cpp

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
```

Comment:
This should probably be updated to reflect that it is an external contribution from "The TinyGo Authors". But IANAL to be clear.